src/components/SortingFilter.js

```diff
@@ -0,0 +1,23 @@
+import { Select } from "./Select.js";
+
+export class SortingFilter {
+  #select;
+
+  constructor({ id, name, options, onChange }) {
+    this.#select = new Select({
+      id,
+      name,
+      options,
+      onChange,
+      className: "restaurant-filter",
+    });
+  }
+
+  get element() {
+    return this.#select.element;
+  }
+
+  get sorting() {
+    return this.#select.value;
+  }
+}
```

Comment:
class로 표현해도 좋지만, 이렇게 함수로 표현해주면 어땠을까 싶어요!

```suggestion
const SortingFilter = (props) => new Select({
  ...props,
  className: "restaurant-filter",
})
```

이러면 getter를 만들어주지 않아도 똑같이 사용할 수 있답니다!